portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -38,14 +37,9 @@
 @WebServlet("/delete-data")
 public class DeleteDataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.getWriter().println("[]");
-  }
-
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Comment").addSort("postTime", SortDirection.DESCENDING);
+    Query query = new Query("Comment").addSort("postTime", SortDirection.DESCENDING).setKeysOnly();
```

Comment:
nit:  if you're deleting everything, probably don't need to sort before hand for efficiency